### ISSUE TITLE
DefaultTreeExpander "Parameter specified as non-null is null" IDE Error

### ISSUE THREAD

**Issue by lfielke** [OpenLiberty/open-liberty-tools-intellij#57]
When viewing and navigating a project, sporadically IntelliJ will show a balloon saying an IDE Error has occurred.

I have the Open Liberty Tools (0.0.5) plugin installed and enabled. The opened IntelliJ project doesn't have any OpenLiberty stuff in it (Gradle plugin not applied, no server selected in run configurations, etc), so I'm not sure if the plugin is assuming something should be there that isn't. We have a number of Java and Kotlin projects that don't use OpenLiberty, so ideally the plugin shouldn't cause IDE Errors in this case.

The info shown by IntelliJ about the error:

```
update failed for AnAction(io.openliberty.tools.intellij.actions.RunLibertyDevTask, id=io.openliberty.tools.intellij.actions.RunLibertyDevTask) with text=Run Liberty action

java.lang.NullPointerException: Parameter specified as non-null is null: method com.intellij.ide.DefaultTreeExpander.<init>, parameter tree
	at com.intellij.ide.DefaultTreeExpander.<init>(DefaultTreeExpander.kt)
	at io.openliberty.tools.intellij.actions.LibertyToolbarActionGroup.<init>(LibertyToolbarActionGroup.java:21)
	at io.openliberty.tools.intellij.LibertyExplorer.buildActionToolbar(LibertyExplorer.java:64)
	at io.openliberty.tools.intellij.LibertyExplorer.<init>(LibertyExplorer.java:57)
	at io.openliberty.tools.intellij.LibertyDevToolWindowFactory.createToolWindowContent(LibertyDevToolWindowFactory.java:15)
	at com.intellij.openapi.wm.impl.ToolWindowImpl.createContentIfNeeded$intellij_platform_ide_impl(ToolWindowImpl.kt:524)
	at com.intellij.openapi.wm.impl.ToolWindowImpl.getContentManager(ToolWindowImpl.kt:385)
	at io.openliberty.tools.intellij.actions.RunLibertyDevTask.update(RunLibertyDevTask.java:29)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.lambda$performDumbAwareUpdate$0(ActionUtil.java:130)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:144)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.doUpdate(ActionUpdater.java:592)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$updateActionReal$7(ActionUpdater.java:132)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$callAction$9(ActionUpdater.java:187)
	at com.intellij.openapi.progress.ProgressManager.lambda$runProcess$0(ProgressManager.java:57)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:183)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:705)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:647)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:63)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:170)
	at com.intellij.openapi.progress.ProgressManager.runProcess(ProgressManager.java:57)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$callAction$10(ActionUpdater.java:185)
	at com.intellij.openapi.actionSystem.impl.ActionUpdateEdtExecutor.lambda$computeOnEdt$0(ActionUpdateEdtExecutor.java:45)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:199)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:794)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$invokeLater$4(ApplicationImpl.java:350)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:84)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:133)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:46)
	at com.intellij.openapi.application.impl.FlushQueue$FlushNow.run(FlushQueue.java:189)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:885)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:754)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:441)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:825)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:440)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:794)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:486)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

IntelliJ info:

> IntelliJ IDEA 2021.2.3 (Ultimate Edition)
> Build #IU-212.5457.46, built on October 12, 2021
> Runtime version: 11.0.12+7-b1504.40 amd64
> VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
> Linux 5.15.12-zen1-1-zen
> GC: G1 Young Generation, G1 Old Generation
> Memory: 3548M
> Cores: 24
> Non-Bundled Plugins: open-liberty.intellij (0.0.5), GrepConsole (12.0.211.6086.4), org.jetbrains.kotlin (212-1.6.10-release-923-IJ5457.46)
> Kotlin: 212-1.6.10-release-923-IJ5457.46
> Current Desktop: XFCE

**Comment by yeekangc** [comment]
Thanks, @lfielke.  @kathrynkodama is taking a look.

**Comment by kathrynkodama** [comment]
Thank you for bringing this to our attention @lfielke. The fix will be included with `0.0.6` of the Open Liberty Tools for IntelliJ plugin. That should be available on the IntelliJ marketplace in the next day or so.

Please do not hesitate to open issues with any other feedback you may have.
